### ISSUE TITLE
New Concept: Binary systems and Multiple systems (Solar System Astronomy)

### ISSUE THREAD

**Issue by stlibrary8** [astrothesaurus/UAT#307]
Binary systems 
Multiple systems
as child term to "Small solar system bodies"

Solar system astronomy
...Solar system
......Small solar system bodies
.........Binary systems 
(propose added _Use For: Binaries (Small solar system bodies)_)
.........Multiple systems
(propose added _Use For: Multiples (Small solar system bodies)_)

Recommended by STScI planetary/solar System researchers.

More recent development in the literature, study of interacting small SS bodies:
https://ui.adsabs.harvard.edu/abs/2017MNRAS.472.3999A/abstract
https://ui.adsabs.harvard.edu/abs/2014acm..conf..285K/abstract
https://ui.adsabs.harvard.edu/abs/2014DPS....4621314C/abstract
https://ui.adsabs.harvard.edu/abs/2016A%26A...595A..20Z/abstract

**Comment by katieefrey** [comment]
I am worried about the danger of these being confused with binary/multiple _star_ systems, or any other system of two (or more) items.

A search for "binary system" in ADS is giving me a wide variety of results about systems of two objects, everything from galaxies, to hot jupiters, to circumbinary stars, to black holes.... many of which use the phrase "binary system" to describe their pairing.

I see "binary asteroids" in the title of one of your linked articles, and I really like that, though I also think that is too specific for this topic... but "Binary small solar system bodies" is way too clunky.

**Comment by stlibrary8** [comment]
I considered something like "Interacting systems" or "Interacting bodies" but then it is very forced. Base don ADS, this doesn't seem to to be how the actual concept is talked about in the literature. It is ultimately your call, and I think it will help to get insight from a journal editor, but I am going to push back a little bit and make the point that any polyhierarchical vocab runs the risk of confusion. I don't see it as a problem to use terms like "Binaries" and "Multiples" that can have multiple meanings since the placement of the concept within the vocab gives the user a clue as to which concept (as opposed to the vague term/word form) they need. 

We saw the same with the "Protoplanetary disks" example. Clarifying with _Use For:_ and making sure the term sits in the right places(s) in the thesaurus is the best we can do I assume to avoid confusion.If researchers state they need a way to describe the concept and this is the most natural way they would talk about and write about it, then it is worth adding, with journal editor input. Editors may have some ideas of how else to describe the concept in a way that is not vague and not weirdly constructed. I agree "binary asteroids" is too specific and "binary small solar system bodies" doesn't make sense. There are no papers that actually use that longer form of the term. 

In short, this can wait till the 2020 release if needed. Hoping a AAS editor can provide more insight.

**Comment by katieefrey** [comment]
> I don't see it as a problem to use terms like "Binaries" and "Multiples" that can have multiple meanings since the placement of the concept within the vocab gives the user a clue as to which concept (as opposed to the vague term/word form) they need.

It wasn't the "binaries" or the "multiples" that gave me pause, but the "system," which is feels generic.  Out of context, it could be a system of stars, or a system of galaxies, etc.

I want to think deeper about how/if the hierarchy informs a concept; since this can wait till the next release, I'd prefer to leave this open for now.

**Comment by katieefrey** [comment]
Looking at this again, I was reminded of it because of an "Asteroid satellites" concept that was proposed...

"Binary systems" by itself still feels too ambiguous.  Hierarchy location is supposed to inform a concepts context, but... concepts should also be able to stand alone.  We saw this exemplified with the "Chaos" concept.  Its context clarified that the concept referred to a KBO, but in all articles it was used, authors clearly meant "unpredictability". 

Asteroid satellites?
Minor planet moons?
https://en.wikipedia.org/wiki/Minor-planet_moon
Not sure if these are related


Binary minor planet systems?
Multiple minor planet systems?

Binary minor planets?
Multiple minor planets?

However, based on guidance from the IAU, I'd like to shy away from using "minor planets."

Q: Is the term minor planet still to be used?
A: The term "minor planet" may still be used. But generally the term small Solar System body will be preferred. 
https://www.iau.org/public/themes/pluto/


Even if an exact phrase may not be used in literature, if the concept described is useful I'm inclined to go with something clunky for now.  It allows the concept to be used and for less clunky phrasing to be suggested in the future.

How about:
Binary SSSB systems
- NPT: Binary small solar system body systems, Binary minor planets, Binary minor planet systems, Binary systems
Multiple SSSB systems
- NPT: (similar to above)
